docs/man/docker.1.md

```diff
@@ -292,79 +292,192 @@ inside it)
 
 # STORAGE DRIVER OPTIONS
 
-Options to storage backend can be specified with **--storage-opt** flags. The
-only backend which currently takes options is *devicemapper*. Therefore use these
+Docker uses storage backends (or "graphdriver" in Docker internals) to create
+writable containers from images.  Many of these backends use operating system
+level technologies and can be configured.
+
+Specify options to the storage backend with **--storage-opt** flags. The only
+backend that currently takes options is *devicemapper*. Therefore use these
 flags with **-s=**devicemapper.
 
+Specifically for devicemapper, the default is a "loopback" model which
+requires no pre-configuration, but is extremely inefficient.  Do not
+use it in production.
+
+To make the best use of Docker with the devicemapper backend,
+you must have a recent version of `lvm2`.  Use `lvm` to create a thin
+pool.  Then, use `--storage-opt dm.thinpooldev` to tell the Docker
+engine to use that pool for allocating images and container snapshots.
+
 Here is the list of *devicemapper* options:
 
+#### dm.thinpooldev
+
+Specifies a custom block storage device to use for the thin pool.
+
+If using a block device for device mapper storage, it is best to use
+`lvm2` to create and manage the thin-pool volume. This volume is then
+handed to Docker to create snapshot volumes needed for images and
+containers.
+
+Managing the thin-pool outside of Docker makes for the most feature-rich method
+of having Docker utilize device mapper thin provisioning as the backing storage
+for Docker's containers. The highlights of the lvm2-based thin-pool management
+feature include: automatic or interactive thin-pool resize support, dynamically
+changing thin-pool features, automatic thinp metadata checking when lvm2 activates
+the thin-pool, etc.
+
+Example use: ``docker -d --storage-opt dm.thinpooldev=/dev/mapper/thin-pool``
```

Comment:
Wondering why you're using double back-ticks here (and some other places below)? 

It could well be valid Markdown, but I'm not familiar with it. GitHub seems to render this correctly, but have you checked if the man-page is generated correctly?